src/nvim/main.c

```diff
@@ -1159,10 +1096,6 @@ static void command_line_scan(mparm_T *parmp)
           }
           break;
 
-        case 'v':                 /* "-v"  Vi-mode (as if called "vi") */
-          exmode_active = 0;
```

Comment:
This probably wasn't used much, so I didn't make it a no-op. What do you think @elmart?